nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -519,6 +519,11 @@
        Please use the fork <literal>cawbird</literal> instead which has been adapted to the API changes and is still maintained.
      </para>
    </listitem>
+   <listitem>
+     <para>
+      The <literal>nodejs-11_x</literal> package has been removed as it's EOLed by upstream.
```

Comment:
This has to be updated for master. We should be targeting 20.03 now.

Reply:
So in your opinion this shouldn't land in 19.09? The thing is, NodeJS 11 is actually EOLed (see also  #69008).

Reply:
No, entirely not my intention.

I just think we should deprecate on master first and then backport it to whever we want it to be. If those target branches then are an upcoming release those should receive an appropriate changelog entry.

If we would merge this now the changelog (from a master perspective) would not reflect the reality of 19.09. That is why I am saying we should update this PR to 20.03. After this has been accepted into 19.09 (by the RMs) then we should go ahead and remove the changelog entry from 20.03 since it would then no longer be true.

We could also drop the message here and only add it to the backport PR. I do not like that since we do not know ahead of time if the RMs would like this kind of change this late in the process.

That all being said. I think our RMs are very reasonable and will happily accept this even at this time.

Reply:
+1 for backporting right away, it sucks but shipping EOL software sucks more :)

Reply:
@andir as we should backport this to 19.09, I guess it should be fine to keep this in the 19.09 release notes, right? (Just talked to @lheckemann about this in person, so unless there are any further objections, I guess it should be fine to merge :))

Reply:
sigh, I wish I had nixos users to talk to :(. Sad times in the states :/